### PR TITLE
Shpejr

### PR DESCRIPTION
**Pull request by techmin**
-Centered layout of the whole page, 
-committee member form link
- officer's email for contact(one officer only)
-short description about SHPEjr

### CODE REVIEW

File: pages/shpejr.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+type Props = {
+  jrName: string;
+  jrEmail: string;
+ 
+};
+
+
+const SHPEJR = ({jrName,jrEmail}:Props) => ( 
+<Layout title="SHPEjr">
```
Comment (gabriel-flynn) [inline pages/shpejr.tsx]:
```suggestion
<Layout title="SHPE Jr.">
```

File: pages/shpejr.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    <div className="has-text-link-dark has-text-centered is-size-1 has-text-weight-bold"
+      style={{marginBottom:".7em"}}
+    >
+      SHPEjr
```
Comment (gabriel-flynn) [inline pages/shpejr.tsx]:
```suggestion
      SHPE Jr.
```

File: pages/shpejr.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+	   dedicated to both the high schoolers and their parents. For the students,we have sessions with 
+	   professional engineers and engineering workshops.At the same time, events are held for the parents, 
+     answering their questions and discussing the benefits and easability of their children going into higher education and STEM.
+     </p>
+     </div>
+   <header 
+      className = "is-size-4"
+      style={{color:"blue",marginLeft:"42%",marginBottom:"1em"}}
+    >
+        Promote STEM in your HIgh School!
```
Comment (gabriel-flynn) [inline pages/shpejr.tsx]:
```suggestion
       Help promote STEM in local High Schools!
```

File: pages/shpejr.tsx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    <div className="has-text-link-dark has-text-centered is-size-1 has-text-weight-bold"
+      style={{marginBottom:".7em"}}
+    >
+      SHPEjr
+    </div>
+<div
+  className="container"
+>     
+     <p
+      style ={{textAlign:"center", marginBottom:".5em", marginLeft:"20%",marginRight:"20%",fontSize:"18px"}}
```
Comment (gabriel-flynn) [inline pages/shpejr.tsx]:
```suggestion
      style ={{textAlign:"justified", marginBottom:".5em", marginLeft:"20%",marginRight:"20%",fontSize:"18px"}}
```

File: pages/shpejr.tsx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+      </ul>
+   
+      <ul
+        className="actions  level-right"
+      //  style={{marginRight:"20%"}}
+      > 
+      <a
+          href={`mailto:${jrEmail}`}
+        >
+        ,<button
```
Comment (gabriel-flynn) [inline pages/shpejr.tsx]:
```suggestion
        <button
```

**Review by gabriel-flynn — CHANGES_REQUESTED** [review]
Looks really good! I requested a few minor changes. One thing I didn't include but I think would help a lot is centering the two buttons and the heading above the two buttons with the title - its a little off center right now at least on my screen.

Also adding a picture below the title might help to add more content to the page, this is the image we currently use on the homepage: http://shpeutd.org/images/pic10.jpg

Let me know if you have any questions/don't agree with some of the suggested changes - wasn't sure on what the best wording would be for some things so just put what I thought might be a little clearer, thanks for your time working on this Atmin!

File: pages/shpejr.tsx [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+>     
+     <p
+      style ={{textAlign:"center", marginBottom:".5em", marginLeft:"20%",marginRight:"20%",fontSize:"18px"}}
+     >
+     At SHPE UTD, SHPE Jr is our high school outreach program, where we work with local SHPE Jr chapters 
+   	 to organize engineering projects, provide academic/professional guidance, and help students foster 
+	   a successful future in STEM.The Noche de Ciencias is our premier event, where a whole evening is 
+	   dedicated to both the high schoolers and their parents. For the students,we have sessions with 
+	   professional engineers and engineering workshops.At the same time, events are held for the parents, 
+     answering their questions and discussing the benefits and easability of their children going into higher education and STEM.
```
Comment (gabriel-flynn) [inline pages/shpejr.tsx:30]:
```suggestion
     &emsp;&emsp;At SHPE UTD, SHPE Jr is our high school outreach program, where we work with local SHPE Jr chapters 
   	 to organize engineering projects, provide academic/professional guidance, and help students foster 
	   a successful future in STEM. The Noche de Ciencias is our premier event, where a whole evening is 
	   dedicated to both the high schoolers and their parents. For the students, we have sessions with 
	   professional engineers and engineering workshops. At the same time, events are held for the parents, 
     answering their questions and discussing the benefits and feasability of their children going into higher education and STEM.
```

Comment (gabriel-flynn) [inline pages/shpejr.tsx]:
```suggestion
     &emsp;&emsp;At SHPE UTD, SHPE Jr is our high school outreach program, where we work with local SHPE Jr chapters 
   	 to organize engineering projects, provide academic/professional guidance, and help students foster 
	   a successful future in STEM. The Noche de Ciencias is our premier event, where a whole evening is 
	   dedicated to both the high schoolers and their parents. For the students, we have sessions with 
	   professional engineers and engineering workshops. At the same time, events are held for the parents, 
     answering their questions and discussing the benefits and feasability of their children going into higher education and STEM.
```

**Review by gabriel-flynn — DISMISSED** [review]

**Comment by techmin** [comment]
made changes manually and giving new request at May 16

**Comment by rigrergl** [comment]
Awesome progress, I think all the button and text looks really good. Only thing I recommend at the moment is that you center the image. Also make sure to merge with the most current version of dev, let me know if you need help resolving any conflicts.

**Comment by techmin** [comment]
I have completed the page the image is centered and the button having better responsiveness in larger resolution.

File: pages/shpejr.tsx [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+      <div
+        className="column is-half is-center is-vcentered"
+        style={{margin:"auto", textAlign:"center"}}
+    >
+
+        <header
+          className="is-size-4"
+        //  style={{color:"blue",marginLeft:"45%",marginBottom:"1em"}}
+        >
+          Help promote STEM in local High School!
```
Comment (rigrergl) [inline pages/shpejr.tsx]:
```suggestion
          Help promote STEM in local High Schools!
```

**Review by rigrergl — APPROVED** [review]
Looks awesome, I would just change "Help promote STEM in local High School!" to "Help promote STEM in local High Schools!"

**Pull request merged**